lib/timeline/component/item/Item.js

```diff
@@ -27,6 +27,7 @@ class Item {
     this.selected = false;
     this.displayed = false;
     this.groupShowing = true;
+    this.selectable = false;
```

Comment:
Shouldn't this be defaulted by the `options.selectable`?

Reply:
Yes but I think it won't make a difference. I'll add it.